shell/platform/tizen/tizen_vsync_waiter.cc

```diff
@@ -4,40 +4,103 @@
 
 #include "tizen_vsync_waiter.h"
 
+#include <eina_thread_queue.h>
+
 #include "flutter/shell/platform/tizen/tizen_embedder_engine.h"
 #include "flutter/shell/platform/tizen/tizen_log.h"
 
-void TizenVsyncWaiter::RequestVblank(void* data, Ecore_Thread* thread) {
-  TizenVsyncWaiter* tizen_vsync_waiter =
-      reinterpret_cast<TizenVsyncWaiter*>(data);
-  if (!ecore_thread_check(thread)) {
-    tdm_error error = tdm_client_vblank_wait(tizen_vsync_waiter->vblank_, 1,
-                                             TdmClientVblankCallback, data);
-    if (error != TDM_ERROR_NONE) {
-      FT_LOGE("tdm_client_vblank_wait error  %d", error);
-      return;
-    }
-    tdm_client_handle_events(tizen_vsync_waiter->client_);
-  }
-}
+static const int QUEUE_QUIT = -1;
+static const int QUEUE_REQUEST_VBLNAK = 0;
+
+typedef struct {
+  Eina_Thread_Queue_Msg head;
+  int value;
+} Msg;
+
+static Eina_Thread_Queue* vblankThreadQueue{nullptr};
```

Comment:
```suggestion
static Eina_Thread_Queue* vblank_thread_queue{nullptr};
```
Naming convention (also applies to `vblankThread_`)